ADR/0038-integration-service-composite-removal.md

```diff
@@ -0,0 +1,31 @@
+# 38. Integration service removes composite snapshots and logic around them
+
+* Date started: 2024-07-10
+
+## Status
+
```

Comment:
You can put accepted into the status. It won't be "officially" accepted until the PR is merged.